### ISSUE TITLE
failed to find pattern

### ISSUE THREAD

**Issue by senmao1994** [R3nzTheCodeGOD/R3nzSkin#82]
![image](https://user-images.githubusercontent.com/50578587/161410764-4cba7cdb-d3c7-4fa7-88e6-623fe21d2a05.png)
How can I use it

**Comment by W1nehazy** [comment]
This is the highest resolution screenshot ive ever saw. Holy fuck.

How is it so clean?

**Comment by souregi** [comment]
I am having the same error on the PBE server. I have updated 3 sigs but it seems that there is something else to change other than updating those 3 sigs.

**Comment by R3nzTheCodeGOD** [comment]
@souregi yes, riot has also changed the **CharacterStackData** structure. Here are the things you need to change to be able to play on PBE.

New CharacterDataStack::Push : `83 EC ? 53 55 56 57 8B F9 8B 47 04 39 07`
New CharacterDataStack::Update : `E8 ? ? ? ? 5F 5E 5B 83 C4 68 C2 04 00`
New CharacterStackData :
```cpp
class CharacterStackData {
public:
	AString model;
	PAD(0xC);
	std::int32_t skin;
};
```

**Comment by borjitarubi** [comment]
hi guys,i saw you resolved this error,but i have the same issue in the actual patch,and i dont know how to fix it,if you can help me boys.
